change/@telekom-design-tokens-df3e3d2a-15bc-44d0-b2e0-7158619e4236.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix motion value",
```

Comment:
The "type" needs to be `prerelease` so it will increase the alpha.X version and not jump into 1.0. You can change this manually.

Also, for the commet, I would include that it's about the CSS output, something "Fix motion values for CSS output" 🙏

Reply:
done!